### PR TITLE
Add min, max, num piles fields. Add start button. Put piles in table rows. 

### PR DESCRIPTION
**Pull request by daveLambe**
Add buttons for min, max and number of piles. Add start game button. Put piles in table rows.
Also added center tags and changed button colour as it looks prettier this way.

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -5,18 +5,34 @@
...
+        </tr>
+    </table>
+
+    <br><br>
 
     <button onclick="updateGameState()">Make Move</button>
     <button onclick="newGame()">New Game</button>
 
     <script src="/static/game_operations.js" charset="utf-8"></script>
+      </center>
```
Comment (romainrichard) [inline templates/index.html]: Too much indented?

File: templates/index.html [hunk 2]
```diff
@@ -5,18 +5,34 @@
...
-    <div id="pile2"></div>
-    <div id="pile3"></div>
-    <div id="pile4"></div>
+        <h3>
+        Enter Minimum stones per pile <br><input name="desired_min" type="text" size="5"> <br><br>
+        Enter Maximum stones per pile <br><input name="desired_max" type="text" size="5" > <br><br>
+        Enter Number of piles <br><input name="desired_piles" type="text" size="5"> <br><br>
+        </h3>
+
+      <button id="start_button" style="font-size: 100%; height: 60px; width: 80px; background-color: aqua;" class="btn btn-default">Start Game!</button><br><br>
```
Comment (romainrichard) [inline templates/index.html]: That button seems redundant with the `New Game` one.

File: templates/index.html [hunk 3]
```diff
@@ -5,18 +5,34 @@
...
-    <div id="pile0"></div>
-    <div id="pile1"></div>
-    <div id="pile2"></div>
-    <div id="pile3"></div>
-    <div id="pile4"></div>
+        <h3>
+        Enter Minimum stones per pile <br><input name="desired_min" type="text" size="5"> <br><br>
+        Enter Maximum stones per pile <br><input name="desired_max" type="text" size="5" > <br><br>
+        Enter Number of piles <br><input name="desired_piles" type="text" size="5"> <br><br>
+        </h3>
```
Comment (romainrichard) [inline templates/index.html]: The ticket said we should pass those to the backend. Did you put it up for review before getting to that? Are you planning on creating a separate ticket to handle that instead?

File: templates/index.html [hunk 4]
```diff
@@ -5,18 +5,34 @@
     <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
   </head>
   <body>
+  <center>
```
Comment (romainrichard) [inline templates/index.html]: We may want to save styling for when we have all the content.

File: templates/index.html [hunk 5]
```diff
@@ -5,18 +5,34 @@
...
+
+    <table>
+    <tr>
+        <td><div id="pile0"></div></td>
+        <td><div id="pile1"></div></td>
+        <td><div id="pile2"></div></td>
+        <td><div id="pile3"></div></td>
+        <td><div id="pile4"></div></td>
+        </tr>
+    </table>
```
Comment (romainrichard) [inline templates/index.html]: Also styling for later?

File: templates/index.html (hunk 3 above)
Comment (daveLambe) [reply templates/index.html]: Ah, so you read the ticket the same way I did. 
Here's how the events transpired: 
- Dave reads the ticket and thinks the scope includes passing JSON to the backend. 
- Dave tries to get this working, has it close but is scratching his head at not being able to get it working as it should. 
- Monday begins, Dave talks to Jesse about said ticket. 
- Jesse informs Dave that this is not at all in the scope of ticket 'nim-6', that the line talking about passing to the backend was simply a comment on what these input fields and button will be eventually used for. 

As far as I'm aware now, that will be dealt with as part of https://hacksaurz.atlassian.net/browse/NIM-16 and https://hacksaurz.atlassian.net/browse/NIM-15

These are now assigned to @dodecaphonix I believe. 
Does that all sound correct @dodecaphonix  ? 

*TLDR:*
I think that's outside the scope of this ticket. Jesse is handling it

File: templates/index.html (hunk 1 above)
Comment (daveLambe) [reply templates/index.html]: Waaaay too much. Changing

File: templates/index.html (hunk 4 above)
Comment (romainrichard) [reply templates/index.html]: Well now that the backend work is out of this ticket, maybe the styling is mostly what is left and thus we should include it?

Comment (daveLambe) [reply templates/index.html]: That was what I was going for. Didn't wanna overdo it since this will change so much. My long term plan is to use a `Google Material Design` library to make it super pretty.

File: templates/index.html (hunk 2 above)
Comment (romainrichard) [reply templates/index.html]: I don't think you need this button. We already have the info displayed and the needed buttons available.
Drop this button and you should be good to go.

Comment (daveLambe) [reply templates/index.html]: Very well, Removing 🙂
